src/tests/App.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../Redux/configureStore';
+import City from '../Components/City';
+import Home from '../Components/Home';
+import Pollution from '../Components/Pollution';
```

Comment:
- [ ] Please we need more tests. Based on the requirements of this project, the app should have unit tests for several components of the app and some pure functions in Redux code. Also, the app should have integration tests using React testing library.

Reply:
Thank you for the feedback @FrederickMih, I have added additional tests for components. Thank you for making the code better.